exercises/concept/elyses-enchantments/.docs/introduction.md

```diff
@@ -1,12 +1,84 @@
 # Introduction
 
 In Javascript, an array is a list-like structure with no fixed length which can hold any type of primitives or objects, even mixed types.
-The array elements can be accessed by their index.
 
-Example of an array declaration and accessing an element by index:
+To create an array, add elements between square brackets `[]`. To read from the array, put the index in square brackets `[]` after the identifier. The indices of an array start at zero.
+
+For example:
 
 ```javascript
 const numbers = [1, 'two', 3, 'four'];
 numbers[2];
 // => 3
 ```
+
+To retrieve the number of elements that are in an array, use the `length` property:
+
+```javascript
+numbers.length;
```

Comment:
If possible every code example block should be stand-alone/self-sufficient. The goal is that a student could paste just this block into e.g. a browsers JS console and see the result for themselves. In this case that means every code block should start with `const numbers = ...`.

Reply:
Thanks for letting me know. I was wondering if it should be stand-alone or build upon the previous block. I will get this updated.